seed/static/seed/js/controllers/inventory_list_controller.js

```diff
@@ -825,6 +825,17 @@ angular.module('BE.seed.controller.inventory_list', [])
                 return _.includes(selectedRowIds, id);
               });
             },
+            filter_header_string: function () {
+              if ($scope.selected_labels.length) {
+                return [
+                  'Filter Method: "',
+                  $scope.labelLogic,
+                  '", Filter Labels: ',
+                  $scope.selected_labels.map(label => label.name).join(', ')
```

Comment:
I don't personally have a strong preference since this is a "non-traditional" CSV, but I did want to pose the question of whether we wanted to join on a different character set than `, `. 

Here's what it looks like with multiple labels:
<img width="1060" alt="Screen Shot 2021-01-14 at 10 49 08 AM" src="https://user-images.githubusercontent.com/30608004/104629150-55b47c80-5656-11eb-801d-f6eef8c899bd.png">

Reply:
Oh... duh.  Didn't think to change the deliminator after moving it into the CSV.  Thanks!

Reply:
![Uploading Screen Shot 2021-01-14 at 1.28.00 PM.png…]()

Reply:
<img width="582" alt="Screen Shot 2021-01-14 at 1 28 00 PM" src="https://user-images.githubusercontent.com/564231/104645631-869fac00-566c-11eb-8031-1352f242e329.png">

Reply:
I see the quotes added, but it still appears to be splitting (at least in Google Sheets)?
<img width="889" alt="Screen Shot 2021-01-14 at 10 27 39 PM" src="https://user-images.githubusercontent.com/30608004/104685303-ef604600-56b7-11eb-888f-512ff568ceea.png">

[test csv export 2.csv.zip](https://github.com/SEED-platform/seed/files/5818643/test.csv.export.2.csv.zip)

Reply:
Looks like sheets isn't honoring the escaped string quotes.  I've updated the PR to have no commas at all in the filter list.